styleguide/components/71-pages/general/topic-overview-page/topic-overview-page--with-images.twig

```diff
@@ -35,7 +35,8 @@
             "teaser_link_text": topic.teaser_link_text,
```

Comment:
This is now variable 'read more' / 'show more'
Maybe we should take it out of config altogether, it's not supposed to be configurable.